src/utils/time.ts

```diff
@@ -251,3 +253,13 @@ export const getDayInterval = (date: Date | number) => ({
   start: startOfDay(date),
   end: endOfDay(date),
 });
+
+export const isDateLessOrEqualsToday = ({
```

Comment:
please, check my another comment about that function. I find confusing the name of it and a set of arguments which should be passed.
maybe we can create a function that checks only one date? not multiple.

Reply:
renamed function in 60ff2eb